### DISCUSSION TITLE
DSO Object Type - What are the numbers at the end?

### DISCUSSION THREAD

**Discussion by CBaker007** [Stellarium/stellarium#2495] [General]
When you search for a DSO, say C 19 (Cocoon Nebula), it shows the "Type" as shown.  

![2022-06-22_10-09-22](https://user-images.githubusercontent.com/77406892/175117573-73653c52-2403-4838-97a5-1397952cf5ef.png)

It starts with a word description (in this case "cluster associated with nebulosity"), then sometimes an acronym for that (in this case "EN+OCL" for Emission Nebula + Open Cluster), and then usually some numbers (in this case "2, 2, 3"). 

What do those numbers stand for / mean?

**Comment by alex-w** [comment]
See "morphological description" - this is human readable description of morphological classification, which you found in parentheses.

**Reply by gzotti** [reply]
I am afraid this is really missing from the Guide. @alex-w Can you please add a list to either section 4.4.3 (better targeted towards beginning users) or chapter 8 (rather for expert or in sense of "in-depth reading/reference")?
